cylinder/workers/de/de.go

```diff
@@ -3,39 +3,43 @@ package de
 import (
 	"github.com/bandprotocol/chain/v3/cylinder"
 	"github.com/bandprotocol/chain/v3/cylinder/context"
+	"github.com/bandprotocol/chain/v3/cylinder/msg"
 )
 
 var _ cylinder.Worker = &DE{}
 
 // DE is a worker responsible for managing own nonce (DE) that being used for signing process
 type DE struct {
-	context *context.Context
-	workers []cylinder.Worker
+	context   *context.Context
+	workers   []cylinder.Worker
+	receivers []*msg.ResponseReceiver
 }
 
 // New creates a new instance of the DE-related workers.
 // It initializes the necessary components and returns the created DE instance or an error if initialization fails.
 func New(ctx *context.Context) (*DE, error) {
-	return &DE{
-		context: ctx,
-	}, nil
-}
-
-// Start starts the DE-related workers.
-func (de *DE) Start() {
-	updateDE, err := NewUpdateDE(de.context)
+	updateDE, err := NewUpdateDE(ctx)
 	if err != nil {
-		de.context.ErrCh <- err
-		return
+		return nil, err
 	}
 
-	deleteDE, err := NewDeleteDE(de.context)
+	deleteDE, err := NewDeleteDE(ctx)
```

Comment:
name updateWorker/updater, DeleteWorker/deleter?

Reply:
keep it as is @RogerKSI